applications/dashboard/controllers/class.utilitycontroller.php

```diff
@@ -308,6 +311,16 @@ public function update() {
             }
         }
 
+        try {
+            $maxCommentLength = Gdn::config('Vanilla.Comment.MaxLength');
+            if ($maxCommentLength > self::MAX_POST_LENGTH) {
+                saveToConfig('Vanilla.Comment.MaxLength', self::MAX_POST_LENGTH);
+            }
+        } catch (Exception $ex) {
+            $this->setData('Success', false);
+            $this->setData('Error', $ex->getMessage());
+        }
+
         if (Gdn::session()->checkPermission('Garden.Settings.Manage')) {
```

Comment:
This is not where this belongs. You've added it to the meta stuff that controls the running of utility/update, instead of adding it to the to-do list. You probably want the `structure()` method in the hooks file of the Vanilla app.

Reply:
This try/catch block isn't necessary, and I'd say the catch condition is just plain wrong. Did you understand what you were doing here and/or test this failure condition?

Reply:
it's been moved.